README.md

```diff
@@ -153,17 +161,17 @@ k8s-worker
 ## note: if the following variables are not defined, their tasks will be skipped
 ##   - cluster_uuid
 [k8s-master]
-cv01 ansible_host=10.0.0.15
-cv02 ansible_host=10.0.0.16
-cv03 ansible_host=10.0.0.17
+hostname01 ansible_host=10.0.0.15
+hostname02 ansible_host=10.0.0.16
+hostname03 ansible_host=10.0.0.17
 
 [k8s-worker]
-cv04 ansible_host=10.0.0.18 cluster_uuid=7273706d-afd5-44ea-8fbf-901ceb6bef27
-cv05 ansible_host=10.0.0.19 cluster_uuid=7273706d-afd5-44ea-8fbf-901ceb6bef27
+hostname04 ansible_host=10.0.0.18 
+hostname05 ansible_host=10.0.0.19
```

Comment:
using cluster_uuid is a valid use, please keep it on at least one or both examples. If you choose to remove on one, please update the pf9 express template as well, not only the readme.

Reply:
corrected